README.md

```diff
@@ -2,16 +2,20 @@
 
 Source code repo for the pelux.io website
 
-## How to build
+## How to build the web site locally
 
-This website uses [jekyll](https://jekyllrb.com/) to build static HTML. Those are the steps to build it:
+This website uses [jekyll](https://jekyllrb.com/) to build static HTML. You'll need some dependencies to build the site locally. These dependencies are best to pull from upstream since they rely on versions that are known to work together. Install the gems jekyll, bundler, and minima this way;
+
+    sudo gem install jekyll bundler minima
```

Comment:
It was required in my case (Debian Testing). I'm hesitant to remove it until I'm certain its not needed.